Source/WebCore/style/StyleScopeRuleSets.cpp

```diff
@@ -277,6 +277,8 @@ void ScopeRuleSets::collectFeatures() const
     m_pseudoClassInvalidationRuleSets.clear();
     m_hasPseudoClassInvalidationRuleSets.clear();
 
+    m_customPropertyNamesInStyleContainerQueries = std::nullopt;
```

Comment:
Not that important here because there is no path forward to generalise this, but uniform initialisation should be preferred `{ }`